gator_grouper/settings.py

```diff
@@ -168,3 +126,22 @@ def find_or_create_secret_key():
 LOGIN_REDIRECT_URL = "Gatorgrouper-home"
 
 LOGIN_URL = "login"
```

Comment:
`LOGIN_URL` is given twice, as is `LOGIN_REDIRECT_URL` (thrice actually)

Reply:
I didn't manually edit any of the travis file.  The only thing I did was run `travis encrypt SOMEVAR="secretvalue"`.  It must have changed the whole file when I did that.  I can change it back.

Reply:
As for the urls, those were all added by someone else.  I'll get rid of the duplicates.